README.Rmd

```diff
@@ -63,13 +63,39 @@ matched <- match_name(loanbook_demo, ald_demo) %>%
   prioritize()
 ```
 
+### Add Scenario Targets
 * Use `target_sda()` to calculate SDA targets of CO2 emissions.
 
 ```{r}
-target_sda(matched, ald_demo, co2_intensity_scenario_demo)
+target_sda(matched, 
+           ald = ald_demo, 
+           co2_intensity_scenario = co2_intensity_scenario_demo)
 ```
 
-* Use `join_ald_scenario()` to join the matched dataset to the relevant 
+* Use `target_market_share` to calculate market-share scenario targets at the portfolio level:
+
+```{r}
+target_market_share(matched, 
+                    ald = ald_demo, 
+                    scenario = scenario_demo_2020, 
+                    region_isos = region_isos_demo)
+```
+
+* ... or at the company level:
+
+```{r}
+target_market_share(matched, 
+                    ald = ald_demo, 
+                    scenario = scenario_demo_2020, 
+                    region_isos = region_isos_demo,
+                    by_company = TRUE)
+```
+
+### Utility Functions
```

Comment:
I consistently use empty lines around headings and paragraphs. I see that's common in most .Rmd I see out there including tidyverse templates (e.g. `use_readme_rmd()`) and I like it because it reads well.

https://github.com/r-lib/usethis/blob/master/inst/templates/rmarkdown-template.Rmd

Reply:
Done